.changeset/lazy-insects-fix.md

```diff
@@ -0,0 +1,7 @@
+---
+"@hopper-ui/icons-react16": minor
+"@hopper-ui/svg-icons": minor
+"@hopper-ui/icons": minor
+---
+
+Renamed ArrowsUpDown Icon to better represent its look / Added Arrow Out Vertical icon
```

Comment:
you should split this changeset in 2, 1 for the svg-icons, and 1 for the icons.

Edit the file to have a bullet list instead of a /

for svg-icons

- Renamed arrows-up-down-X to maximize-X 
- Added arrow-out-vertical-X.svg

for icons and icons-react16
- Renamed ArrowUpdownIcon to ...
- Added ArrowOut ...

Since we're passing this breaking change as a minor version, we should have a clear documentation